fe/src/component/bottom/Slide.js

```diff
@@ -1,36 +1,73 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 import Card from "../UI/Card";
-import CardsWrapper from "../UI/CardsWrapper";
-import SlideIcon from "./SlideIcon";
+import { SlideIcon } from "../icons/SlideIcon";
+import constansts from "../../constants/constansts";
+
+const SlideWrapper = styled.div`
+  position: relative;
+  height: 460px;
+  display: none;
+  &.active {
+    display: block;
+  }
+`;
+
 const SlideTitle = styled.h5`
-  font-size: 24px;
-  font-weight: 500l;
-  color: #333;
-  margin: 40px 0 0 40px;
+  font-size: ${({ theme }) => theme.fontSize.mLarge};
+  font-weight: ${({ theme }) => theme.fontWeight.bold};
+  color: ${({ theme }) => theme.colors.black};
+  margin: 40px 0 20px 40px;
+`;
+
+const SlideCardWrapper = styled.div`
+  position: absolute;
+  width: 1280px;
+  height: 450px;
+  overflow: hidden;
+  margin-left: 50px;
 `;
 
-const Ts = styled.div`
+const SlideCardsList = styled.div`
   display: flex;
+  transform: translateX(${({ currentTranslateX }) => currentTranslateX});
 `;
-const Slide = () => {
-  const [slideInfo, setSlideInfo] = useState([]);
-  useEffect(() => {
-    fetch("https://api.codesquad.kr/onban/soup")
-      .then((res) => res.json())
-      .then((data) => setSlideInfo(data.body));
-  }, []);
+
+const Slide = ({ cardData, alwaysDisplayed, isDisplayed }) => {
+  const [cardListTranslateX, setCardListTranslateX] = useState(0);
+  const cardSize = constansts.CardSize.small;
+  const maxTranslateX = -(cardData.body.length * cardSize) + 4 * cardSize;
+  const onClickRightHandler = () => {
+    setCardListTranslateX((prev) =>
+      prev - cardSize >= maxTranslateX ? prev - cardSize : prev
+    );
+  };
+
+  const onClickLeftHandler = () => {
+    setCardListTranslateX((prev) =>
+      prev + cardSize <= 0 ? prev + cardSize : prev
+    );
+  };
+
+  const click = (event, props) => {
+    console.log(event.target.id);
+  };
 
   return (
-    <>
-      <SlideTitle>식탁을 풍성하게 하는 정갈한 밑반찬</SlideTitle>;
-      <CardsWrapper>
-        <SlideIcon />
-        <Ts>
-          {slideInfo.map((v) => {
+    <SlideWrapper
+      className={isDisplayed ? "active" : ""}
+      style={{ display: alwaysDisplayed && "block" }}
+    >
+      <SlideTitle>식탁을 풍성하게 하는 정갈한 밑반찬</SlideTitle>
+      <SlideCardWrapper>
+        <SlideCardsList
+          currentTranslateX={cardListTranslateX.toString() + "px"}
```

Comment:
```suggestion
          currentTranslateX={`${cardListTranslateX}px`}
```
template literal을 적극 사용해 봅시다!